project_pivot/__init__.py

```diff
@@ -0,0 +1 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```

Comment:
init files never includes license advertisement (they don't content real code, and less if there's nothing).